torchvision/ops/giou_loss.py

```diff
@@ -36,7 +36,7 @@ def generalized_box_iou_loss(
             ``'none'`` | ``'mean'`` | ``'sum'``. ``'none'``: No reduction will be
             applied to the output. ``'mean'``: The output will be averaged.
             ``'sum'``: The output will be summed. Default: ``'none'``
-        eps (float, optional): small number to prevent division by zero. Default: 1e-7
+        eps (float): small number to prevent division by zero. Default: 1e-7
```

Comment:
Similar to `eps`,  doc-string of `reduction`  parameter contains unnecessary  `optional`. I think you should remove that as well.

Reply:
Noticed, thanks for the pointer. However, I didn't write the `giou_loss.py` file. :smile:

Reply:
yes, I know 😄. 

BTW  I think it's okay if we don't remove the `optional`  from the above docstrings.  Refer - https://github.com/pytorch/vision/pull/5799#discussion_r847518711

Reply:
I see, thanks for the link @abhi-glitchhg. Indeed, here the `eps` argument has a default value so the optional makes some sense (even though it is better to use it for the None case).